### PR TITLE
Remove the about page

### PR DESCRIPTION
**Pull request by henryhadlow**
Show, don’t tell.

### CODE REVIEW

File: app/appBuilder.js [hunk 1]
```diff
@@ -97,8 +97,6 @@ module.exports = {
 
     app.get('/performance', require('./server/controllers/homepage'));
 
-    app.get('/performance/about', require('./server/controllers/about'));
```
Comment (jabley) [inline app/appBuilder.js]: Will need to redirect to something. We can't remove URLs from the internet.

Comment (henryhadlow) [inline app/appBuilder.js]: Ha. Yes. I thought this was a test :$

Comment (henryhadlow) [inline app/appBuilder.js]: Working with @easternbloc to fix this.

Comment (jonnywyatt) [inline app/appBuilder.js]: @easternbloc - does this need more changes to be merge-able?

**Comment by easternbloc** [comment]
@jonnywyatt ¯\_(ツ)_/¯ needs rebase that much I'm aware of

**Comment by easternbloc** [comment]
Closing until it's rebased.

**Pull request closed without merging**